pkg/config/application.go

```diff
@@ -588,14 +588,21 @@ type DeploymentNotification struct {
 	Mentions []NotificationMention `json:"mentions"`
 }
 
-func (n *DeploymentNotification) FindSlackAccounts(event model.NotificationEventType) []string {
+func (n *DeploymentNotification) FindSlackAccountsAndGroups(event model.NotificationEventType) []string {
 	as := make(map[string]struct{})
 	for _, m := range n.Mentions {
 		if m.Event != allEventsSymbol && "EVENT_"+m.Event != event.String() {
 			continue
 		}
-		for _, s := range m.Slack {
-			as[s] = struct{}{}
+		if len(m.Slack) > 0 {
```

Comment:
It would be better if we changed `Slack` to `SlackUsers` to separate it from `SlackGroups`.

Reply:
@minhquang4334 thanks, will have a look

Reply:
how if we keep `Slack` field and mark it as deprecated field and add `SlackUsers`
wdyt? @minhquang4334

Reply:
also if we keep `Slack`, thinking 3 option
- combine 2 those field to final list users mention
- set priority to use new field only
- throw error from validate() that only support one of those at same time
wdyt?

Reply:
> how if we keep Slack field and mark it as deprecated field and add SlackUsers
wdyt? @minhquang4334

LGTM

> set priority to use new field only

IMO, this one seems easier for users to use, and removing the Slack field minimizes the impact of breaking changes 🤔

Reply:
@minhquang4334  thanks for your feed back
update the PR https://github.com/pipe-cd/pipecd/pull/4903/commits/e957946f41e50b145ae5556aef03b4b27f5d1090
please have a look

Reply:
LGTM